test/e2e/framework/job/rest.go

```diff
@@ -20,12 +20,13 @@ import (
 	"fmt"
 
 	batch "k8s.io/api/batch/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
related to https://github.com/kubernetes/kubernetes/issues/77045

Reply:
If possible, I'd like to keep these changes out of this PR until 77045 is resolved. Keeps the PRs easier to read. Maybe switch to `gofmt` instead of `goimports for now?

Reply:
Fine for this PR, but for future PRs that would be great :)